File: dev-docs/snippets/databases/list-permissions.swift
import JsBaoClient

// List all permission entries for a database.
func listPermissions(client: JsBaoClient, databaseId: String) async throws {
  // #region example
  let permissions = try await client.databases.listPermissions(databaseId: databaseId)
  // #endregion example
  _ = permissions
}

File: dev-docs/snippets/databases/list-permissions.ts
import type { JsBaoClient } from "js-bao-wss-client";

// List all permission entries for a database.
export async function listPermissions(client: JsBaoClient, databaseId: string) {
  // #region example
  const permissions = await client.databases.listPermissions(databaseId);
  // #endregion example
  return permissions;
}
